scipy/optimize/zeros.py

```diff
@@ -123,6 +123,33 @@ def newton(func, x0, fprime=None, args=(), tol=1.48e-8, maxiter=50,
     sign. The brentq algorithm is recommended for general use in one
     dimensional problems when such an interval has been found.
 
+    Examples
+    --------
+    >>> def f(x):
+    ...     return (x**3 - 1)  # only one real root at x = 1
+    
+    >>> from scipy import optimize
+    
+    >>> # fprime and fprime2 not provided, use secant method
```

Comment:
It would probably be nicer to write this line before the interpreter section rather than include it as a code comment.